api/envoy/extensions/access_loggers/file/v3/file.proto

```diff
@@ -60,4 +60,7 @@ message FileAccessLog {
     config.core.v3.SubstitutionFormatString log_format = 5
         [(validate.rules).message = {required: true}];
   }
+
+  // The output JSON string properties will be sorted if json_format is used and sort_properties is set to true.
+  bool sort_properties = 6;
```

Comment:
I don't think this is necessary because the `json_format` of `FileAccessLog` is deprecated.

Reply:
Ack